en/starter/generator/getting_started.md

```diff
@@ -18,8 +18,12 @@ redirect_from: "/starter/generator/getting_started.html"
 	<p>Use the <b>kitura create</b> command to create scaffolding for a simple Kitura web application.	</p>
 </div>
 
-> ![warning] Make sure you have installed the command-line tools as described in
-> [Installation](installation.html) before you begin.
+> ![warning] Make sure you have installed the command-line tools and prerequisites as described in
+> [Getting Started](../gettingstarted.html) before you begin.
+
+> ![tip] When accessing the command line remotely, using a terminal emulator such as [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/),
```

Comment:
This seems a pretty niche and unrelated tip... I'm inclined not to include it.

Reply:
...but (as the 'niche' Windows user among us) that was the reason I ended up here in the first place :'(

Reply:
In seriousness though, I agree that it's pretty irrelevant to Mac and Linux users.  Perhaps it belongs in a 'troubleshooting' section instead?  I do seriously expect someone else to trip over it, hence my motivation.